Makefile

```diff
@@ -41,7 +58,11 @@ docker/weave:
 	curl -L git.io/weave -o docker/weave
 	chmod u+x docker/weave
 
-$(SCOPE_EXPORT): $(SCOPE_EXE) $(DOCKER_DISTRIB) docker/weave $(RUNSVINIT) docker/Dockerfile docker/demo.json docker/run-app docker/run-probe docker/entrypoint.sh
+docker/ebpf.tgz: Makefile
+	$(SUDO) docker pull $(EBPF_IMAGE)
+	CONTAINER_ID=$(shell $(SUDO) docker run -d $(EBPF_IMAGE) /bin/false 2>/dev/null || true); $(SUDO) docker export -o docker/ebpf.tgz $${CONTAINER_ID}
```

Comment:
That's more or less what I did :)

I used an elf section to add the following:
```
char _build_id[] SEC("build_id") = CIRCLE_BUILD_URL;
```

compiled with clang with option `-DCIRCLE_BUILD_URL=\"$(CIRCLE_BUILD_URL)\"`

Then, the Scope build and the tcptracer-bpf build will both print the build URL:

```
	sha1sum ebpf/ebpf.o
	objdump -s --section=build_id ebpf/ebpf.o | grep "^ " | xxd -r | tr "\0" "\n"
```

So the build logs on CircleCI will keep that information.

Should we update this PR as long as we go, or address all review tasks and update it in one go? At the moment, @iaguis has commits in another branch to avoid too much traffic on this PR page:
https://github.com/kinvolk/scope/commits/iaguis/conn-perf-ebpf-guess-ci

Do you want other information than `$CIRCLE_BUILD_URL`?
I could add the commit id `$CIRCLE_SHA1` too, although it is visible in the CircleCI webpage already.